Number_types/include/CGAL/Sqrt_extension/Sqrt_extension_type.h

```diff
@@ -676,6 +678,17 @@ CGAL::Comparison_result
     { return ( p.compare(num) == CGAL::LARGER ); }
 };
 
+template <class NT, class ROOT_, class ACDE_TAG_, class FP_TAG >
+inline bool operator <  (const Sqrt_extension<NT, ROOT_, ACDE_TAG_, FP_TAG>& p, const NT& num)
+{
+    return (p.compare(num) == CGAL::SMALLER);
+}
+
+template <class NT, class ROOT_, class ACDE_TAG_, class FP_TAG >
+inline bool operator >  (const Sqrt_extension<NT, ROOT_, ACDE_TAG_, FP_TAG>& p, const NT& num)
+{
+    return (p.compare(num) == CGAL::LARGER);
+}
```

Comment:
Note that `inline` is redundant with because those are already templates.

Reply:
fixed. thank you.